src/test/java/dev/jbang/cli/TestRun.java

```diff
@@ -115,7 +115,7 @@ void testHelloWorld(boolean first) throws IOException {
 
 		String result = new JarCmdGenerator(code, ctx).generate();
 
-		assertThat(result, startsWith("java "));
+		assertThat(result.split(" ")[0], endsWith("java"));
```

Comment:
Same reason, because of the change in how to find the default JRE it can now happen that instead of just being plain "java" it suddenly becomes "/some/path/to/a/jdk/bin/java".

Reply:
Ok, I'm removing that commit again. Making the tests work on Windows just makes things overly complicated. I'll just have to fix my dev environment :-)
Perhaps some other time we can come up with something better.